server/src/server.rs

```diff
@@ -138,6 +141,11 @@ impl Server {
                 .ok_or(NotFound::NodeBySlot(slot))?;
             (src_node, dest_node)
         };
+        if let Err(e) = src_node.forwarding_limiter.check() {
```

Comment:
Could you use `check_n` to limit the amount of bytes (`proto.payload.len()`) that can be sent per second (e.g. 600 B/s or 36000/min)?